.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "c:\\DevOps\\Project_Exercises\\DevOps-Course-Starter\\.venv\\Scripts\\python.exe"
```

Comment:
You shouldn't commit things which are specific to your computer.
In this case, this can be a relative path `"python.pythonPath": ".venv\\Scripts\\python.exe"`